framework/pom.xml

```diff
@@ -147,6 +147,20 @@
 			<version>0.9.30</version>
 			<scope>test</scope>
 		</dependency>
+
+
+		<!-- https://mvnrepository.com/artifact/org.postgresql/postgresql -->
+		<dependency>
```

Comment:
Je nutná v tomto prípade zavislost na konkretnu databazu? neriesi to Hikari? Ak nie, tak je potreba tam pridat vsetko co sonarqube podporuje.